python/ray/dashboard/utils.py

```diff
@@ -692,3 +692,9 @@ def compose_state_message(
         else:
             state_message = death_reason_message
     return state_message
+
+
+def close_logger_file_descriptor(logger_instance):
+    for handler in logger_instance.handlers:
+        handler.close()
+        logger_instance.removeHandler(handler)
```

Comment:
I think the issue is still we are removing the handlers here. This will cause any log statement after this is called to a logger becoming a no-op